Model/Api/OrderManagement.php

```diff
@@ -218,23 +220,24 @@ public function manage(
      * @param $reference
      * @param $type
      * @param $display_id
+     * @param $immutable_quote_id
      * @param $storeId
      * @throws \Bolt\Boltpay\Exception\BoltException
      */
-    private function saveUpdateOrder($reference, $type, $display_id, $storeId)
+    private function saveUpdateOrder($reference, $type, $display_id, $immutable_quote_id, $storeId)
```

Comment:
Adding new parameters is better done at the end of the list of existing ones. Usually a good practice, not relevant here, though.

Reply:
good to know for future. My thinking was since I was splitting `display_id` it should go with, but I'll remember that for next time